src/passes/I64ToI32Lowering.cpp

```diff
@@ -29,6 +29,8 @@
 #include "wasm-builder.h"
 #include "ir/names.h"
 
+#define ROTL64_NAME "__wasm_rotl_i64"
```

Comment:
these appear to already be defined in `src/asmjs/shared-constants.h` which we could include here